### PR TITLE
alt text and basic accessibility adds

### PR DESCRIPTION
**Pull request by moates**
Had to add a new template param to include alt text. Everything compiled for me (except I couldn't test Micah & Shafali's very well because relative links were broken for me locally). 

Still todos:

- [X]  hamburger menu

- [X] angelica's gifs
- [ ] footnotes

### CODE REVIEW

**Comment by moates** [comment]
@bennlich agreed to take on the footnote accessibility! ty!

File: layouts/partials/prev-and-next-buttons.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
-<div class="prev-and-next-buttons">
+<div class="prev-and-next-buttons" role="navigation">
   
   {{/*
-    writing code in go template syntax is totally lame. The code
+    writing code in go template syntax is :( The code
```
Comment (bennlich) [inline layouts/partials/prev-and-next-buttons.html:4]: :)

**Comment by bennlich** [comment]
Wowwiiiiiiiiiieeeeee thank you @moates !!!! I appreciate this work so very much! :pray: 

@benhylau @uditvira @maisutton  If we had insider web-aim nerd-shirts, I think they would say `alt="worm"`.

**Comment by bennlich** [comment]
Really appreciate the alt-text on the interactive elements. I've never seen that done before.

**Pull request merged**